modules/nw-aws-nlb-existing-cluster.adoc

```diff
@@ -50,7 +50,7 @@ spec:
           type: NLB
 ----
 <1> Replace `$my_ingress_controller` with a unique name for the Ingress Controller.
-<2> Replace `$my_unique_ingress_domain` with a domain name that is unique among all Ingress Controllers in the cluster.
+<2> Replace `$my_unique_ingress_domain` with a domain name that is unique among all Ingress Controllers in the cluster. This variable must be a sub-domain of the DNS name `<clustername>.<domain>`.
```

Comment:
"Subdomain" does not require hyphenation. :slightly_smiling_face:

Reply:
Thanks, fixed this.